financialaid/admin.py

```diff
@@ -20,6 +20,7 @@ class FinancialAidAdmin(admin.ModelAdmin):
 class FinancialAidAuditAdmin(admin.ModelAdmin):
     """Admin for FinancialAidAudit"""
     model = FinancialAidAudit
+    readonly_fields = ["acting_user", "financial_aid", "data_before", "data_after"]
```

Comment:
Since the goal is to make _all_ fields read-only, it's better to dynamically get the list of fields from the model, instead of hard-coding them here. You can get the fields from a model by calling `Model._meta.get_fields()`. So in this case, the code would be:

```python
readonly_fields = [f.name for f in FinancialAidAudit._meta.get_fields()]
```

When I ran that locally, I got the following list of fields: `['id', 'created_on', 'updated_on', 'acting_user', 'financial_aid', 'data_before', 'data_after']` That's more than you have listed here -- is that an intended difference?